### ISSUE TITLE
Support for corporate proxy ?

### ISSUE THREAD

**Issue by johanpoirier** [desmondmorris/node-twitter#10]
Hi, I can't use this module behind a corporate proxy. Is it possible to add proxy conf to the http module ?

**Comment by desmondmorris** [comment]
@kauegimenes @johanpoirier @darul75 I am [just about finished](https://github.com/desmondmorris/node-twitter/tree/request) with replacing the oauth module in favor of [request](http://github.com/request/request), which has a oauth signing built in and proxy support.

The idea is that you will be able to pass request options to the Twitter constructor.  Something like:

````
var client = new Twitter({
  request_options: {
    "proxy": "http://my.proxy.server"
  }
});
````

The `request_options` object gets passed directly to request.  Hopefully this will be helpful.

**Comment by desmondmorris** [comment]
This support has shipped in v1.2.0.  I tested it locally with squidman and it seems to be working.
